app/src/main/java/org/gdg/frisbee/android/onboarding/FirstStartStep2Fragment.java

```diff
@@ -62,6 +102,39 @@ public void onSkippedSignIn() {
         listener.onSkippedSignIn();
     }
 
+    public void loadSender(String inviteSender) {
+        this.inviteSender = inviteSender;
+        if (!isContextValid()) {
+            return;
+        }
+
+        App.getInstance().getPlusApi()
+            .getPerson(inviteSender)
+            .enqueue(new Callback<Person>() {
+                @Override
+                public void success(Person sender) {
```

Comment:
If we fail to load the sender then the invitation process stops here. We should still show the inviteContainer with a dummy image and a dummy name ("a friend" or so)